### DISCUSSION TITLE
metabase Dosen't connecting in clickhouseDB

### DISCUSSION THREAD

**Discussion by Mohammed-Bilal-Stitch** [ClickHouse/metabase-clickhouse-driver#186] [Q&A]
this Error how to solve :
 No suitable driver found for jdbc:clickhouse://http://localhost:8123/nginxdb default

pls help for this issue

**Comment by enqueue** [comment]
Are you using 1.2.0 driver release for a _non_ 0.47 Metabase installation?

**Comment by Mohammed-Bilal-Stitch** [comment]
Yes I am using updated version.

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by enqueue** [reply]
This is not supported. See [Release Info](https://github.com/ClickHouse/metabase-clickhouse-driver/releases/tag/1.2.0):

> Important: Metabase 0.47.x is required

You should use the prior [version 1.1.7](https://github.com/ClickHouse/metabase-clickhouse-driver/releases/tag/1.1.7).

**Comment by Mohammed-Bilal-Stitch** [comment]
Yes , working 😇 .Thank you
From: Felix Mueller ***@***.***>
Date: Thursday, 10 August 2023 at 10:02 PM
To: ClickHouse/metabase-clickhouse-driver ***@***.***>
Cc: Mohammed Bilal SB ***@***.***>, Author ***@***.***>
Subject: Re: [ClickHouse/metabase-clickhouse-driver] metabase Dosen't connecting in clickhouseDB (Discussion #186)

This is not supported. See Release Info<https://github.com/ClickHouse/metabase-clickhouse-driver/releases/tag/1.2.0>:

Important: Metabase 0.47.x is required

You should use the prior version 1.1.7<https://github.com/ClickHouse/metabase-clickhouse-driver/releases/tag/1.1.7>.
